tox.ini

```diff
@@ -68,11 +68,14 @@ setenv =
     OBJC_DISABLE_INITIALIZE_FORK_SAFETY = YES
 whitelist_externals =
     tests,functests,bddtests: sh
-commands =
-    dev: {posargs:pserve conf/development.ini --reload}
+    dev,tests,functests,bddtests: make
+commands_pre =
+    {dev,tests,functests,bddtests}: make -C {toxinidir} services
```

Comment:
This is the part we'd need to skip for jenkins. Could just be a case of sh `$JENKINS_VAR || make...` or something?